config/betterquesting/DefaultQuests.json

```diff
@@ -3929,7 +4193,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§oYes, you will need more Steam production!§r\n\nThe Steam Turbine will help you convert Steam into Electricity. Or so that\u0027s the simplest explanation I can think of...\n\n§d2 Steam \u003d 1 EU§r\n\nThis one Generator is the only one you can realistically use in LV Age. Later you will unlock more power options.\n\n\n§9Reminder:§r Any machine placed can be rotated with a Wrench. To retrieve a machine (even an electric one), you can simply mine it. Wrenches work, but pickaxes also do.",
+          "desc:8": "§oYes, you will need more Steam production!§r\n\nThe Steam Turbine will help you convert Steam into Electricity. Or so that\u0027s the simplest explanation I can think of...\n\n§d2 Steam \u003d 1 EU§r\n\nThis one Generator type is the only one you can realistically use in LV Age. Later you will unlock more power options.\n\n\n§9Reminder:§r Any machine placed can be rotated with a Wrench. To retrieve a machine (even an electric one), you can simply mine it. Wrenches work, but pickaxes also do.\n\n\n§oQuest rewards will become less generous from now on.§r",
```

Comment:
```suggestion
          "desc:8": "§oYes, you will need more Steam production!§r\n\nThe Steam Turbine will help you convert Steam into Electricity. Or so that\u0027s the simplest explanation I can think of...\n\n§d2 Steam \u003d 1 EU§r\n\nThis one Generator type is the only one you can realistically use in the LV Age. Later you will unlock more power options.\n\n\n§9Reminder:§r Any machine placed can be rotated with a Wrench. To retrieve a machine (even an electric one), you can simply mine it. Wrenches work, but pickaxes also do.\n\n\n§oQuest rewards will become less generous from now on.§r",
```